data/test-cases/plugins/ms-teams-sync/attachments-sync/Allowed_attachment_size.md

```diff
@@ -0,0 +1,51 @@
+---
+# (Required) Ensure all values are filled up
+name: "Allowed size of attachment"
+status: Active
+priority: Normal
+folder: attachments-sync
+authors: "@AayushChaudhary0001"
+team_ownership: []
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Click on the attachments button in channel or DM/GM in MM.
```

Comment:
```suggestion
1. Upload and send an attachment of the allowed size in the linked channel or DM/GM in MM.
```

Reply:
maybe?

Reply:
Yeah, we can club the selecting steps for the attachment as "uploading the attachment" and for sending the attachment i think we can define it in separate step.